### ISSUE TITLE
No configuration methods succeeded

### ISSUE THREAD

**Issue by wsy0101** [siderolabs/sidero#1202]
(1) My bois setting is UEFI IPv6 PXE WORK startup

(2) DHCPv6 settings:

Subnet6 ffaa::/64{

Range6 ffaa:: 5 ffaa:: 100;

Default easy time 600;

Max lease time 720;

Option dhcp6. bootfile url "tftp://[fec0:0:2601:1:: 1082]/ipxe. efi";

}

(3) Then the machine started up and obtained an assigned ipv6 address from dhcpv6. Then, it went to the tftp of sidero metal to obtain the boot file of ipxe. efi and found the following error:

Waiting for link up on net2... ok

Configuring (net2 20: ce: 2a: 71: da; e9) No configuration methods succeeded（ https://ipxe.org/040ee186 ）

May I ask how I need to set up to start using uefi on ipv6 and obtain the configuration method so that it can be taken over by sidero metal?

**Comment by smira** [comment]
I don't think that iPXE Sidero Metal is using is built with DHCPv6, neither the Sidero Agent supports DHCPv6.

Talos Linux itself is IPv6 ready, but not Sidero Metal.

**Comment by wsy0101** [comment]
Do Sidero Metal have plans to include DHCPv6 in the subsequent iteration versions？

**Comment by smira** [comment]
There are no specific plans at the moment.

**Comment by wsy0101** [comment]
这是来自QQ邮箱的假期自动回复邮件。
 
您好，我最近正在休假中，无法亲自回复您的邮件。我将在假期结束后，尽快给您回复。

**Comment by wsy0101** [comment]
这是来自QQ邮箱的假期自动回复邮件。
 
您好，我最近正在休假中，无法亲自回复您的邮件。我将在假期结束后，尽快给您回复。

**Comment by wsy0101** [comment]
这是来自QQ邮箱的假期自动回复邮件。
 
您好，我最近正在休假中，无法亲自回复您的邮件。我将在假期结束后，尽快给您回复。
